src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java

```diff
@@ -94,7 +95,7 @@ private Object readResolve() {
     }
 
     @Override
-    public void onLoad(Run<?, ?> build) {
+    public void onLoad(@NonNull Run<?, ?> build) {
```

Comment:
Better to add annotations in the defining method in core.

Reply:
done https://github.com/jenkinsci/jenkins/pull/3543